projects/samples/contests/robocup/protos/RobocupSoccerField.proto

```diff
@@ -13,257 +13,245 @@ PROTO RobocupSoccerField [
 {
 
 %{
--- Official soccer field dimensions from RoboCup 2021 (in meters)
+  -- Official soccer field dimensions from RoboCup 2021 (in meters)
 
-  LW = 0.05 -- Line width
-  MS = 0.1 -- Length of one branch of the penalty/center mark (NOT SPECIFIED IN RULES)
-  n = 36 -- Number of vertices for the regular polygon approximating the central circle (NOT SPECIFIED IN RULES)
+  line_width = 0.05 -- Line width
+  branch_length = 0.1 -- Length of one branch of the penalty/center mark (not specified in rules)
+  n_vertices_circle = 36 -- Number of vertices for the regular polygon approximating the central circle (not specified in rules)
 
   if fields.size.value == "kid" then
-    -- KidSize dimensions
-    A = 9
-    B = 6
-    C = 0.6
-    D = 2.6
-    E = 1
-    F = 3
-    G = 1.5
-    H = 1.5
-    I = 1
-    J = 2
-    K = 5
+      -- KidSize dimensions
```

Comment:
Why is this indented 4 spaces instead of 2?
It seems Lua is commonly using 2 spaces.

Reply:
Thank you Stefania, I have corrected this !